source/administration/concepts.rst

```diff
@@ -0,0 +1,162 @@
+MinIO supports S3-specific :ref:`actions <minio-policy-actions>` and :ref:`conditions <minio-policy-conditions>` when creating policies. 
+
+By default, MinIO *denies* access to actions or resources not explicitly referenced in a user's assigned or inherited policies.
+
+MinIO provides an access management feature as part of the software.
+Alternatively, you can configure MinIO to authenticate with one of several external IAM providers using either :ref:`Active Directory/LDAP <minio-external-identity-management-ad-ldap>` or :ref:`OpenID/OIDC <minio-external-identity-management-openid>`.
+
+
+How does MinIO secure data?
+---------------------------
+
+MinIO supports methods that encode objects while on disk (encryption-at-rest) and during transition from one location to another (encryption-in-transit, or "in flight").
+When enabled, MinIO utilizes :ref:`server-side encryption <minio-encryption-overview>` to write objects in an encrypted state.
+To retrieve and read an encrypted object, the user must have appropriate access privileges and also provide the object's decryption key.
+
+MinIO supports **Transport Layer Security** (TLS) versions 1.2 and 1.3 encrypting objects.
+TLS replaces the previously used Secure Socket Layer (SSL) method that has since been deprecated.
+The TLS standard, maintained by the Internet Engineering Task Force (IETF), provides the standards used by internet communications to support encryption, authentication, and data integrity.
+
+The process of authenticating a user and verifying access to objects is known as the `TLS Handshake`.
+Once authenticated, TLS provides the cipher to encypt and then decrypt the transfer of information from the server to the requesting client.
+
+MinIO supports several methods of :ref:`Server-Side Encryption <minio-encryption-overview>`.
+
+Can I organize objects in a folder structure within buckets?
+------------------------------------------------------------
+
+MinIO is a flat object store, meaning there is no structure to a deployment other than the bucket(s).
+However, MinIO does allow for using a prefixing scheme with a ``/`` delimitor that can mimic a folder structure from traditional file systems.
+
+Prefixing involves prepending the name of an object with a fixed string.
+When you use the same prefix string on multiple objects, MinIO identifies those as similar or grouped objects.
+
+Consider an object in an ``images`` bucket named ``vacation/2019/october/17/mountain-hike.jpg``.
+MinIO treats this as an object ``mountain-hike.jpg`` in a nested series of locations:
+
+.. code-block::
+
+   vacation/
+      2019/
+         october/
+            17
+
+If you had a second object in the bucket named ``vacation/2019/october/17/waterfall.jpg``, you could find both objects by calling on the ``vacation/2019/october/17/`` prefix.
+
+Remember:
+
+- MinIO buckets do not actually arrange contents in folders.
```

Comment:
Not sure what this section is intended to get across.

Reply:
cut